packages/components/src/FormatFile/FormatFile.tsx

```diff
@@ -32,46 +75,132 @@ export function FormatFile({ file, onDelete }: FormatFileProps) {
   }
 
   const style = imageSource ? { backgroundImage: `url(${imageSource})` } : {};
+  const imageBlockStyle = isComplete
+    ? styles.imageBlock
+    : classNames(styles.imageBlock, styles.imageBlockOverlay);
 
   return (
-    <div className={styles.formatFile}>
-      <div className={styles.imageBlock} style={style} data-testid="imageBlock">
+    <div
+      onMouseEnter={() => {
+        showDeleteButtonIfSmallThumbnail(
+          display === "compact" && displaySize === "default",
```

Comment:
Maybe move this to a const 
`const isSmallThumbnail =  display === "compact" && displaySize === "default"`
 and reuse it